### ISSUE TITLE
[Feature Request] Realtime Hybrid mode

### ISSUE THREAD

**Issue by mikeloeven** [Gorea235/TaledensInvManagerUpdated#16]
I would like to request a real-time hybrid mode. Essentially a execution method that utilizes both real-time delays and interval update. 

Basically using Tick intervals results in good response times but bad performance while intervals that are 5-10 seconds long cause problems when the script needs to split jobs up into multiple cycles. 

My proposal is a hybrid system that will use the real time delay for normal operations and the update interval for when the script needs to split up it's execution 

For example I will enable hybrid execution, set real time interval to 10 seconds and the tick interval to 10 ticks

what this will do is execute the script every 10 seconds if the script cannot finish in a single execution it will run each subsequent step every 10 ticks until complete and than wait 10 seconds for the next execution. This way when the script has to be broken up into multiple cycles you will not need to wait 40 - 50 seconds for a single execution to complete. 

Basically you run with an inner and outer loop so that the sub-cycles are completed in a timely manner without putting excessive load on the server.
